src/dialects/oracledb/index.js

```diff
@@ -53,7 +53,7 @@ Client_Oracledb.prototype.columnCompiler = function() {
   return new ColumnCompiler(this, ...arguments)
 }
 Client_Oracledb.prototype.formatter = function() {
-  return new Oracledb_Formatter(this)
+  return new Oracledb_Formatter(this, ...arguments)
```

Comment:
Shouldn't all these be 

```
= function(builder) {
   return new Oracledb_Formatter(this, builder);
}
```

Thats more understandable for anyone reading the code.

Reply:
I had it like that at first but then I noticed that the convention was passing all the arguments, for example on line 53 there:https://github.com/tgriesser/knex/blob/59f6cba17839435e3c10d9e1011d5b78e4a970f0/src/dialects/oracledb/index.js#L53

Or is that because those are the generic compiler and transaction classes?

I agree though that it's more readable..

Reply:
I suppose it has just been personal preference of the one who originally implemented oracledb dialect.

Reply:
Hmm.. but then it's also the same on the [mssql dialect](https://github.com/joelmukuthu/knex/blob/cdb91b8f8759ae6f40f1007070996cdf31797014/src/dialects/mssql/index.js#L49-L67). I don't mean to prod I'm just a sucker for conventions :)

Reply:
Both dialects are pretty new ones and might have been used as a reference for each other. But sure you are right, since that syntax is used everywhere in those dialects, there is no reason why it couldn't be used also in this PR. So no need to change that.